<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(241, 241, 241);</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="full_top_to_bottom_layout" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="top_header_frame_blue" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(56, 145, 255);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>33</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>ïŽ•</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>MobyUI</string>
            </property>
            <property name="margin">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="middle_frame" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="nav_widget_left" native="true">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="nav_widget_left_layout">
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <item>
              <widget class="QWidget" name="nav_buttons" native="true">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QAbstractButton {
	height: 40px;
	background-color: transparent;
    
	border: 0;
    border-left: 10px solid transparent;
	border-top: 1px solid transparent;
	border-bottom: 1px solid transparent;
	border-right: 1px solid transparent;
	
	text-align: left;
	text-color: black;
	padding-left: 30px;
	qproperty-iconSize: 25px;
}

QAbstractButton:hover {
	border-left-color: transparent;
	border-top-color: transparent;
	border-bottom-color: transparent;

	background-color: rgb(218, 234, 255);
}

QAbstractButton:checked {
	border-left-color: rgb(56, 145, 255);
	border-top-color: rgb(56, 145, 255);
	border-bottom-color: rgb(56, 145, 255);
	border-right-color: rgb(56, 145, 255);
	
	background-color: rgb(209, 230, 255);
}</string>
               </property>
               <layout class="QVBoxLayout" name="nav_buttons_layout">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QToolButton" name="nav_button_containers">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>   Containers</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../res/icons.qrc">
                    <normaloff>:/icons/fontawesome/black/cubes.svg</normaloff>:/icons/fontawesome/black/cubes.svg</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="nav_button_images">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>   Images</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../res/icons.qrc">
                    <normaloff>:/icons/fontawesome/black/layer-group.svg</normaloff>:/icons/fontawesome/black/layer-group.svg</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="nav_button_volumes">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>   Volumes</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../res/icons.qrc">
                    <normaloff>:/icons/fontawesome/black/hard-drive.svg</normaloff>:/icons/fontawesome/black/hard-drive.svg</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="nav_button_compose">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>   Docker Compose</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../res/icons.qrc">
                    <normaloff>:/icons/fontawesome/black/docker.svg</normaloff>:/icons/fontawesome/black/docker.svg</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="nav_button_registries">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>    Registries</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../res/icons.qrc">
                    <normaloff>:/icons/fontawesome/black/database.svg</normaloff>:/icons/fontawesome/black/database.svg</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="content_widget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="bottom_footer_frame_white" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-top: 1px solid;
border-top-color: rgb(222, 221, 218);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>30</number>
          </property>
          <item>
           <widget class="QWidget" name="daemon_status_widget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 0;</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="docker_daemon_status_logo">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(224, 27, 36);</string>
               </property>
               <property name="text">
                <string>ïŽ•</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="docker_daemon_status_text">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Docker daemon not running</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>nav_button_containers</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedNavContainers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nav_button_images</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedNavImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nav_button_volumes</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedNavVolumes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nav_button_compose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedNavCompose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nav_button_registries</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedNavRegistries()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clickedNavContainers()</slot>
  <slot>clickedNavImages()</slot>
  <slot>clickedNavVolumes()</slot>
  <slot>clickedNavCompose()</slot>
  <slot>clickedNavRegistries()</slot>
 </slots>
</ui>
